automated-comments/csharp/general/convert_variable_to_const.md

```diff
@@ -0,0 +1 @@
+You can convert the `%{name}` variable to a `const` as the value is intended never to change. You can either declare the `const` a method-only constant, by defining it within the method (similar to how the variable is defined), or make it a class-only constant, which is the more idiomatic solution.
```

Comment:
@mikedamay said:

“make it a class-only constant, which is the more idiomatic solution”. I had never thought of a class level constant being more idiomatic than method level. I would say that the decision depends on the role of the value in the code – a class level constant where it reflects business rules or some real-world value or method level where a number of fixed values are used in different combinations for some reason."

Reply:
@mikedamay These additions are great. Do you perhaps have a suggestion for a new comment?

Reply:
You can convert the `%{name}` variable to a `const` as the value is intended never to change.  You will most often see `const`s defined at the class level as they are frequently used in multiple methods.  You have the option of defining them at the method level when their use is restricted to a single method.  This minimises the scope within which they are visible.

FYI but not part of the comment
https://stackoverflow.com/questions/6373072/defining-local-variable-const-vs-class-const

Reply:
I like it! I've updated the comment.